src/DataCollector/ModelsCollector.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Barryvdh\Debugbar\DataCollector;
+
+use Barryvdh\Debugbar\DataFormatter\SimpleFormatter;
+use DebugBar\DataCollector\MessagesCollector;
+use Illuminate\Events\Dispatcher;
+use Illuminate\Support\Str;
+
+/**
+ * Collector for Models.
+ */
+class ModelsCollector extends MessagesCollector
+{
+    public $models = [];
+
+    /**
+     * @param Dispatcher $events
+     */
+    public function __construct(Dispatcher $events)
+    {
+        parent::__construct('models');
+        $this->setDataFormatter(new SimpleFormatter());
```

Comment:
is this being used somewhere else?

Reply:
Yes...I stole it from other collectors. For example:

https://github.com/barryvdh/laravel-debugbar/blob/bda348f9105a3cdd6dd9bd3966298b69fdd1dfcc/src/DataCollector/GateCollector.php#L19-L22